jbpm-designer-client/src/main/resources/org/jbpm/designer/public/defaults/WorkDefinitions.wid.st

```diff
@@ -56,6 +56,15 @@ import org.drools.core.process.core.datatype.impl.type.ObjectDataType;
     ],
     "displayName" : "REST",
     "icon" : "defaultservicenodeicon.png"
+  ],
+
+  [
+    "name" : "Milestone",
+    "parameters" : [
+      "Condition" : new StringDataType()
+    ],
+    "displayName" : "Milestone",
+    "icon" : "defaultmilestoneicon.png"
```

Comment:
just one comment - not sure milestone should be added to all none case projects as that's what we do when console creates new case project to provide special types there instead. But if we see a point in having it by default in all projects that's ok to.